### ISSUE TITLE
Spurious Travis-CI build failures due to tests hanging

### ISSUE THREAD

**Issue by pschord** [psforever/PSF-LoginServer#60]
Check out https://travis-ci.org/psforever/PSF-LoginServer/builds/152131655 and https://travis-ci.org/psforever/PSF-LoginServer/builds/152123698

I have noticed this hang when testing locally as well. It's something to do with the native PSCrypto library (but not the library code itself). Can't reproduce at the moment

**Comment by Fate-JH** [comment]
I will verify that this has been happening to me locally, sometimes, when running the tests too.  I have a short rundown in the Discord/code channel but what happens is that the tests for a certain group will complete, e.g., all of the GamePacketTest tests, then it may stop and wait indefinitely on a blank line preceded by [info].  Ctrl+c out of it.  Like the order of tests run/completed , it's not very consistent.

In regards to the crypto, I intentionally killed the tests before they started running, as that scrolls the intro wherein the compiling and setup is explained in the command prompt out of the buffer, and found this message:
```
net.psforever.crypto.CryptoInterface.CryptoDHState: class not closed. memory leaked
```
It happens everytime.   (The longer list of tests is due to local changes.  Removing that test file does not change the message.)

**Comment by pschord** [comment]
Regarding `class not closed` that's okay to have in the tests as we dont care about memory leaks. If we _ever_ see that in the actual server, that's bad.

Hmmm, interesting that Specs seems to be hanging like that. Will have to investigate but it's definitely thread related.

**Comment by Fate-JH** [comment]
Should be "fixed" by PR #96 and PR #100.
